wiki/People/Quality_Assurance_Team/en.md

```diff
@@ -15,21 +15,25 @@ Team Members
 
-| Name                                              | osu!         | osu!taiko    | osu!catch    | osu!mania    | Additional Languages       |
-|---------------------------------------------------|:------------:|:------------:|:------------:|:------------:|----------------------------|
-| [Asherz007](https://osu.ppy.sh/users/9014047)         | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] |                            |
-| [Deif](https://osu.ppy.sh/users/318565)               | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] | Spanish, German            |
-| [Doyak](https://osu.ppy.sh/users/2046893)             | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Korean                     |
-| [Fycho](https://osu.ppy.sh/users/1876867)             | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Chinese                    |
-| [JBHyperion](https://osu.ppy.sh/users/4879508)        | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] |                            |
-| [Kurai](https://osu.ppy.sh/users/77089)               | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | French                     |
-| [Kurokami](https://osu.ppy.sh/users/260933)           | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] | Hungarian                  |
-| [Lasse](https://osu.ppy.sh/users/896613)              | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | German                     |
-| [Mao](https://osu.ppy.sh/users/2204515)               | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | German                     |
-| [Myxomatosis](https://osu.ppy.sh/users/2202645)       | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | German                     |
-| [Naxess](https://osu.ppy.sh/users/8129817)            | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Swedish                    |
-| [Nardoxyribonucleic](https://osu.ppy.sh/users/876419) | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | Chinese, Cantonese         |
-| [p3n](https://osu.ppy.sh/users/123703)                | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | German                     |
-| [pishifat](https://osu.ppy.sh/users/3178418)          | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                            |
-| [Protastic101](https://osu.ppy.sh/users/6712747)      | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] |                            |
-| [Raiden](https://osu.ppy.sh/users/2239480)            | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | Spanish, Catalan           |
+| Name                                              | osu!         | osu!taiko    | osu!catch    | osu!mania    | Additional Languages       | Primary Branch       |
+|---------------------------------------------------|:------------:|:------------:|:------------:|:------------:|----------------------------:|----------------------------:|
```

Comment:
Why use right align for the last two columns?

Reply:
that's' how it was like, but moved left